feconf.py

```diff
@@ -202,6 +202,15 @@
 
 # Default name for the initial state of an exploration.
 DEFAULT_INIT_STATE_NAME = 'Introduction'
+
+DEFAULT_NEW_STATE_CONTENT_ID = 'content'
+
+DEFAULT_OUTCOME_CONTENT_ID = 'default_outcome'
+
+DEFAULT_CONTENT_IDS_TO_AUDIO_TRANSLATIONS = {
+    'content': {},
```

Comment:
Maybe should just use the constants above as the key names?

Reply:
I'll do this after finalizing the structure of NEW_STATE_TEMPLATE.